Ensure credentials are not being stored or captured

JCantu248 opened the pull request:
# <!-- Use the title to describe PR changes in the imperative mood --> #

## 🗣 Description ##

<!-- Describe the "what" of your changes in detail. -->
<!-- To avoid scope creep, limit changes to a single goal. -->

Minor revision to configure all campaigns to not allow either credentials or passwords to be captured by Gophish.

## 💭 Motivation and context ##

<!-- Why is this change required? -->
<!-- What problem does this change solve? How did you solve it? -->
<!-- Mention any related issue(s) here using appropriate keywords such
<!-- as "closes" or "resolves" to auto-close them on merge. -->

Discussing with Kelly Theile and the PCA team, we discussed if passwords are being stored in plaintext (#52) and we were informed that we are not allowed to receive any credentials in any form (plaintext or encrypted) for legal reasons. Credentials must be dropped by the landing pages. This change ensures that no credential or password data is captured. 

In addition, setting the capture credential and password  boolean values was done in one place, before being repeated on each branch of execution. 

Closes #52 

## 🧪 Testing ##

<!-- How did you test your changes? How could someone else test this PR? -->

<!-- Include details of your testing environment, and the tests you ran to -->
<!-- see how your change affects other areas of the code, etc. -->

This was tested by bringing the code changes into our LiPCA process and checking that the campaigns created afterwards were no longer configured to capture submitted data. After setting capture credentials to false, the checkbox in the Gophish GUI for capture submitted data is no longer set. 


## ✅ Checklist ##

<!-- Remove any of the following that do not apply. -->
<!-- Draft PRs should have one or more unchecked boxes. -->
<!-- If you're unsure about any of these, don't hesitate to ask. -->
<!-- We're here to help! -->

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] _All_ future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] All new and existing tests pass.

JCantu248 committed:
Changing Gophish landing pages to not capture any credentials or password.

Boolean values that control if Gophish captures credentials and passwords has been modified to not capture this data. Credentials and passwords should not be retained by Gophish.

JCantu248 committed:
Bump project version from 0.0.8 to 0.0.9

JCantu248 committed:
Bump project version from 0.0.9 to 0.0.9-rc.1

JCantu248 committed:
Bump project version from 0.0.9-rc.1 to 0.0.10

JCantu248 committed:
Bump project version from 0.0.10 to 0.0.10-rc.1

JCantu248 committed:
Bump project version from 0.0.10-rc.1 to 0.0.10-rc.2

JCantu248 committed:
Cleaning up some blank lines prior to merger

JCantu248 committed:
Bump project version from 0.0.10-rc.2 to 0.0.10

dav3r commented on src/_version.py:
```diff
@@ -1,2 +1,2 @@
 """This file defines the version of this project."""
-__version__ = "0.0.8"
+__version__ = "0.0.10"
```
What happened to version `0.0.9`?  

On a related note, do you have an idea of what has to happen for this project to reach version `1.0.0`?

JCantu248 replied:
I think somewhere along the way, I must have screwed up with the bump_version tool and making a pre-release. I think I might have already used the v0.0.9 tag accidentally.

JCantu248 replied:
Also regarding a v.1, I think we have some Sprint planning we are doing currently for our Live PCA project, and that may define what v.1 of Gophish-tools would look like.

dav3r replied:
According to the rules at [semver.org](https://semver.org/), this should probably be a minor release, meaning you could bump it to version `0.1.0` if you wanted.

JCantu248 replied:
I bumped it to v0.1.0 in 4a392778cc9f97d4950443b2a11aa44ca6d64237 

Additionally, I merged the latest changes from develop into this feature branch.

dav3r reviewed (approved):
This looks good to me, though please see my questions about versions.

dav3r commented:
You should also merge in the latest changes from `develop` to ensure there are no conflicts.

JCantu248 committed:
Bump project version from 0.0.10 to 0.1.0

JCantu248 committed:
Merging latest changes for develop into Remove_credentials_password_gathering prior to merging pull request #54.

Merge branch 'develop' of github.com:cisagov/gophish-tools into Remove_credentials_password_gathering

jsf9k reviewed (approved):
:+1:
